### DISCUSSION TITLE
Volume² 1.1.9.466

### DISCUSSION THREAD

**Discussion by irzyxa** [irzyxa/Volume2#312] [General]
Added ability to add devices to the favorite list. This list is used for tray context menu and Set Next/Prev Default Device function ([#311](https://github.com/irzyxa/Volume2/issues/311))
Added support for horizontal mouse scroll ([#309](https://github.com/irzyxa/Volume2/issues/309))
Clicking on tray icon again while mixer is open (if left button single click set to volume mixer) should close the mixer ([#298](https://github.com/irzyxa/Volume2/issues/298))

<hr /><em>This discussion was created from the release <a href='https://github.com/irzyxa/Volume2/releases/tag/v1.1.9.466'>Volume² 1.1.9.466</a>.</em>

**Comment by temetor** [comment]
This version scores poorly in the sandbox on Hybrid Analysis:
https://www.hybrid-analysis.com/sample/2bcc7457b1c6790f37795f753ff225c829a483bc3bf7a8ceec7c00d773b42d17

Can you confirm why is that?
